pom.xml

```diff
@@ -139,6 +139,7 @@
     <slf4j.version>1.7.2</slf4j.version>
     <test.output.redirect>true</test.output.redirect>
     <hadoop-openstack.version>2.6.0</hadoop-openstack.version>
+    <yarn.version>2.6.0</yarn.version>
```

Comment:
how does this interact/is different from the `yarn.version` set in the yarn profile?